camayoc/tests/qpc/conftest.py

```diff
@@ -2,9 +2,19 @@
 import pytest
 
 from camayoc import api
+from camayoc.data_provider import DataProvider
 from camayoc.tests.qpc.utils import sort_and_delete
 
 
+@pytest.fixture(scope="session")
```

Comment:
I made this session-scoped, as opposed to function-scoped cleanup, to remove everything in bulk at the end of test run. On my local environment (camayoc local and quipucords local, both from git), that gave significant performance improvement - tests completed about 30% faster.

On CI, API tests complete about 5% faster, which is nice, but rather inconsequential.